src/Account.jsx

```diff
@@ -23,7 +23,7 @@ class Account extends React.Component {
       repayAmount: "",
       repayMessage: "",
       repayRecepient: "",
-      time: 300,
+      time: 3000,
```

Comment:
What is "time" used for? You should come up with a more descriptive name for this. Does it even belong in the state?

Reply:
Done.